packages/account/src/Components/poi/idv-status/idv-submit-complete/idv-submit-complete.tsx

```diff
@@ -1,26 +1,55 @@
-import IdvDocumentPending from 'Assets/ic-idv-document-pending.svg';
-import PoaButton from 'Components/poa/poa-button';
 import React from 'react';
 import { Text } from '@deriv/components';
+import { idv_error_statuses, TIDVErrorStatus } from '@deriv/shared';
 import { localize } from '@deriv/translations';
+import IdvDocumentPending from 'Assets/ic-idv-document-pending.svg';
+import PoaButton from 'Components/poa/poa-button';
 
 type TIdvSubmitComplete = {
-    needs_poa: boolean;
     is_from_external: boolean;
-    redirect_button: React.ReactElement;
+    mismatch_status: TIDVErrorStatus;
+    needs_poa: boolean;
+    redirect_button: React.ReactNode;
 };
 
-const IdvSubmitComplete = ({ needs_poa, is_from_external, redirect_button }: TIdvSubmitComplete) => {
+const IdvSubmitComplete = ({
+    is_from_external,
+    mismatch_status,
+    needs_poa,
+    redirect_button,
+}: Partial<TIdvSubmitComplete>) => {
+    const is_mismatch_error =
```

Comment:
this line can be reduced to a single line like 
`const  { poi_name_dob_mismatch, poi_dob_mismatch, poi_name_mismatch } = idv_error_statuses;
 const is_mismatch_error = [poi_name_dob_mismatch, poi_dob_mismatch, poi_name_mismatch].includes(mismatch_status);
`